qemu/tests/cfg/qemu_img_convert_from_vdi.cfg

```diff
@@ -0,0 +1,16 @@
+- qemu_img_convert_from_vdi:
+    only RHEL
+    only i440fx
+    no Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
+    required_qemu = [6.1.0-7, )
+    virt_test_type = qemu
+    type = qemu_img_convert_from_vdi
+    start_vm = no
+    kill_vm = yes
+    images = "rhel850"
+    image_name_rhel850 = images/rhel850
+    image_format_rhel850 = vdi
```

Comment:
images = vdiimg
image_format_vdiimg = vdi

Set image_name_vdiimg in py file:
params['image_name_vdiimg'] = os.path.basename(params["vdi_image_url"]).split('.')[0]

vdi_image_address -> vdi_image_url

I am also OK with the current, thanks.

Reply:
I agree with Zhenchao, rhel850 in the private cfg is OK.

Reply:
Thanks.